app/views/posts/index.html.erb

```diff
@@ -1 +1,24 @@
-<h1> Hello List of posts will appear right here</h1>
\ No newline at end of file
+<%= render 'shared/user_card', user: @users, user_card_class: 'user_card no_hover' %>
+
+<ul class='post_list'>
+  <% if @users.posts.empty? %>
+    <p class='user_empty-message'>No posts yet!</p>
+  <% else %>
+    <% @users.most_recent_three_posts.each do |post| %>
+      <li class='post_list_item'>
+        <%= render 'shared/post_card', post: post %>
+        <ul class='post_comment_list'>
+          <% if post.comments.empty? %>
+            <p class='comment_empty-message'>No comments yet!</p>
+          <% else %>
+            <% posts.most_recent_five_comments.each do |comment| %>
```

Comment:
Please note that when I click on the `See all posts` button, I'm met with the below error. I believe it has to do with line 14, where you should have `post.` and not `posts.` 😉
![Screenshot from 2023-07-19 22-49-32](https://github.com/KUBAHO3/blog-app/assets/44624138/71ca9666-a9d3-42f7-8f64-5ed89889cf54)